phpstan-baseline.neon

```diff
@@ -4490,6 +4490,11 @@ parameters:
 			count: 1
 			path: src/Sylius/Bundle/UserBundle/Form/UserVerifiedAtToBooleanTransformer.php
 
+		-
+			message: "#^Interface Sylius\\\\Bundle\\\\UserBundle\\\\Provider\\\\UserProviderInterface extends generic interface Symfony\\\\Component\\\\Security\\\\Core\\\\User\\\\UserProviderInterface but does not specify its types\\: TUser$#"
+			count: 1
+			path: src/Sylius/Bundle/UserBundle/Provider/UserProviderInterface.php
+
```

Comment:
Specify the `TUser` type via `@extends` PHPDoc.

Reply:
Nvm, `5.4` build is complaining about declaring a generic type while extending the interface without a generic type : (.